README.md

```diff
@@ -8,7 +8,64 @@ Check out this GitHub example of the Syncfusion .NET MAUI Busy Indicator control
 https://github.com/syncfusion/maui-demos/tree/master/MAUI/BusyIndicator
 
 ## Project prerequisites
-Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/ ) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
+Make sure that you have the compatible versions of [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/) with the Dot NET MAUI workload and [.NET Core SDK 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) or later version in your machine before starting to work on this project.
 
 ## How to run this application
-To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-busy-indicator` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
\ No newline at end of file
+To run this application, you need to first clone the `getting-started-with-the-dotnet-maui-busy-indicator` repository and then open it in Visual Studio 2022. Now, simply build and run your project to view the output.
+
+## <a name="troubleshooting"></a>Troubleshooting ##
+### Path too long exception
+If you are facing a path too long exception when building this example project, close Visual Studio rename the repository to short, and build the project.
+
+## License
+
+Syncfusion has no liability for any damage or consequence that may arise by using or viewing the samples. The samples are for demonstrative purposes, and if you choose to use or access the samples, you agree to not hold Syncfusion liable, in any form, for any damage that is related to use, for accessing, or viewing the samples. By accessing, viewing, or seeing the samples, you acknowledge and agree Syncfusion’s samples will not allow you seek injunctive relief in any form for any claim related to the sample. If you do not agree to this, do not view, access, utilize, or otherwise do anything with Syncfusion’s samples.

```

Comment:
Remov this sections
![image](https://github.com/SyncfusionExamples/getting-started-with-the-dotnet-maui-busy-indicator/assets/95272441/6448c1dc-1afb-404d-8ec8-dd6168bcbbc9)

Reply:
Section removed.